### ISSUE TITLE
Cannot open Pass in Ios Device after Upgrading to 0.1.1

### ISSUE THREAD

**Issue by BESAFX** [drallgood/jpasskit#253]
The process of signing pass succeeded with no errors and i will attach here 2 different files of same pass using both versions (old version 0.1.0) and (latest version 0.1.1), Please extract Passes.zip file and you will find both pkapss files using old version and new version, Also i checked and validate both generated passes (please use file with extension .pkpass ) using this web service: 
http://pkpassvalidator.com/ and both validated.

[Passes.zip](https://github.com/drallgood/jpasskit/files/4287327/Passes.zip)

**Comment by drallgood** [comment]
I just had a look and the only difference is the date:
```
➜  Passes diff old/pass.json new/pass.json
61c61
<   "expirationDate": "2020-02-01T08:00:00Z",
---
>   "expirationDate": "2020-02-01T08:00:00.000+0000",
```

And indeed it fails due to the date format: 
```
Invalid data error reading pass pass.test.christin.june/d173a38b-ede9-411c-abe1-04b3b8c62a53. Unable to parse expirationDate 2020-02-01T08:00:00.000+0000 as a date. We expect dates in "W3C date time stamp format", either "Complete date plus hours and minutes" or "Complete date plus hours, minutes and seconds". For example, 1980-05-07T10:30-05:00.
```

I think we fixed that in 0.2 but haven't released it yet.

**Comment by drallgood** [comment]
I think I'll release 0.2 as 0.1.1 is apparently broken.
It contains some breaking changes. I hope that's ok?

**Comment by BESAFX** [comment]
Thank you for your quick replay, So when you will release this fix ?

**Comment by drallgood** [comment]
I’ll try to do it later today. 
Just be aware that 0.2 has some significant breaking changes

**Comment by BESAFX** [comment]
That is a big problem ! Is there a way to merge this fix to current release 0.1.1 ?

**Comment by drallgood** [comment]
Should be doable.
However, I strongly recommend upgrading as there were quite a few changes

**Comment by drallgood** [comment]
Released fix as 0.1.2

**Comment by BESAFX** [comment]
@drallgood 
Thank you so mush man, I am really appreciate your hard work on this library :+1: 
Thanks Again :)
